### DISCUSSION TITLE
Amazon.Lambda.APIGatewayEvents Nuget package for .NET 6?

### DISCUSSION THREAD

**Discussion by andrehartmann1968** [aws/aws-lambda-dotnet#1252] [Q&A] [answered]
Hello,

  when will the Amazon.Lambda.APIGatewayEvents Nuget package be available for .NET 6?

  .NET Core 3.1 is nearing end of support in November/December and we want to migrate our .NET Lambda functions to .NET 6. Some of them need the Amazon.Lambda.APIGatewayEvents package but it does not run on a .NET 6 Lambda because it is only for .NET Standard 2.0 and .NET Core 3.1...

  Amazon.Lambda.Core has already been published for .NET 6.

  Thanks in advance!

PS: error message received from Lambda function:
1658386124068,"It was not possible to find any compatible framework version"
1658386124068,"The framework 'Microsoft.NETCore.App', version '3.1.0' (x64) was not found."
1658386124068,'"- The following frameworks were found:"
1658386124068,"6.0.4 at [/var/lang/bin/shared/Microsoft.NETCore.App]"
1658386124068,"You can resolve the problem by installing the specified framework and/or SDK."
1658386124068,"The specified framework can be found at:"
1658386124068,'"- https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=3.1.0&arch=x64&rid=amzn.2-x64"

This error does not occur when I deploy a different Lambda function on .NET 6 that does not use Amazon.Lambda.APIGatewayEvents.

**Comment by normj** [comment, marked as answer]
Amazon.Lambda.APIGatewayEvents already supports .NET 6. There was no specific changes that were required for .NET 6 so we didn't need to add another target framework and .NET will happily use the .NET Core 3.1.
